### ISSUE TITLE
AutoRotation - screen always 90° in wrong direction

### ISSUE THREAD

**Issue by Tropaion** [phhusson/treble_experimentations#2037]
I installad various phhusson based ROMs inclusive the original on my Lenovo Xiaoxin Pad Pro 2021 (J716F) but in every one, when I activate AutoRotate, the screen is 90° in wrong direction.

**Comment by Tropaion** [comment]
I tried solving it by adding "ro.surface_flinger.primary_display_orientation=ORIENTATION_270" to build.prop, the screen rotated in right direction, but the touchpanel did not.​

**Comment by phhusson** [comment]
Yup, if it's auto-rotation, it's not the display orientation you want to change, but accelerometer rotation
There is no facility for this baked in aosp, it'll probably require a specific android change, i'm not sure where

**Comment by Tropaion** [comment]
Yeah, I already suspected that. So there is no way to solve this problem?
I think this isn't to only device where the accelerometer axes are replaced. Is it possible to add an offset somewhere?
